libs/components/ag-grid/src/lib/modules/ag-grid/top-horizontal-scroll/top-horizontal-scroll.service.ts

```diff
@@ -0,0 +1,76 @@
+import { ElementRef, Injectable, OnDestroy } from '@angular/core';
+import { MutationObserverService } from '@skyux/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TopHorizontalScrollService implements OnDestroy {
+  private mutationObserver: MutationObserver;
+
+  constructor(private mutationObserverSvc: MutationObserverService) {}
+
+  public ngOnDestroy(): void {
+    if (this.mutationObserver) {
+      this.mutationObserver.disconnect();
+    }
+  }
+
+  public appendTopScrollbarToGrid(elementRef: ElementRef): void {
+    // css class selectors
+    const headerSelector = '.ag-header';
```

Comment:
I'd move these constants outside of the class since they don't change across calls to this function.

Reply:
Moved to the top of the file.